autopilot/contractor.go

```diff
@@ -726,11 +721,6 @@ func (c *contractor) runContractRenewals(ctx context.Context, w Worker, budget *
 			"budget", budget,
 		)
 	}()
-	// start renewing from the largest contract to lose the least amount of data
-	// in case we have more contracts than we need.
-	sort.Slice(toRenew, func(i, j int) bool {
```

Comment:
This sorting here ruined the sorting we did on a higher level where we also take into account whether the contract is part of the set.